main/registerIpcMainActionListeners.ts

```diff
@@ -180,6 +200,37 @@ export default function registerIpcMainActionListeners(main: Main) {
     return obj;
   });
 
+  ipcMain.handle('download-update-manual', async () => {
+    const availableUpdate = getAvailableUpdate();
+    if (!availableUpdate) {
+      throw new Error('No update available to download');
+    }
+
+    const currentVersion = app.getVersion();
+    const isCurrentBeta = currentVersion.includes('beta');
+    const isNextBeta = availableUpdate.version.includes('beta');
+
+    let downloadUpdate = true;
+
+    if (!isCurrentBeta && isNextBeta) {
+      const option = await dialog.showMessageBox({
+        type: 'info',
+        title: 'Update Available',
+        message: `Download version ${availableUpdate.version}?`,
+        buttons: ['Yes', 'No'],
+      });
```

Comment:
Please check if you could apply the above message regarding title and description can be applied here too.